pymc_experimental/model/model_api.py

```diff
@@ -0,0 +1,46 @@
+from functools import wraps
+
+import pymc as pm
+
+
+def as_model(*model_args, **model_kwargs):
+    R"""
+    Decorator to provide context to PyMC models declared in a function.
+    This removes all need to think about context managers and lets you separate creating a generative model from using the model.
+
+    Inspired by the `sampled <https://github.com/colcarroll/sampled>`_ decorator for PyMC3.
```

Comment:
@zaxtax's blog post was the more direct inspiration here, no? Maybe instead you could link to the blog post.